Logging to shared log directory: /tmp/test_logs//1331862913
only running test 0a
excepting tests: 15c
Starting client mft01: -o user_xattr,acl,flock mft01@tcp:/lustre /mnt/lustre2
Started clients mft01: 
mft01@tcp:/lustre on /mnt/lustre2 type lustre (rw,user_xattr,acl,flock)
debug=-1
subsystem_debug=0xffb7e3ff
debug_mb=2
mft01: Checking config lustre mounted on /mnt/lustre
Checking servers environments
Checking clients mft01 environments
Using TIMEOUT=20
disable quota as required
debug=-1
subsystem_debug=0xffb7e3ff
debug_mb=2
setting all flavor to null
already have total 8 null connections

== replay-dual test 0a: expired recovery with lost client == 03:55:26 (1331862926)
Filesystem           1K-blocks      Used Available Use% Mounted on
mft01@tcp:/lustre       374928     49528    305272  14% /mnt/lustre
total: 50 creates in 0.06 seconds: 850.75 creates/second
fail_loc=0x80000514
Failing mds1 on node mft01
Stopping /mnt/mds1 (opts:) on mft01
affected facets: mds1
Failover mds1 to mft01
03:55:41 (1331862941) waiting for mft01 network 900 secs ...
03:55:41 (1331862941) network interface is UP
Starting mds1: -o loop,user_xattr,acl  /tmp/lustre-mdt1 /mnt/mds1
debug=-1
subsystem_debug=0xffb7e3ff
debug_mb=2
Started lustre-MDT0000
Starting client: mft01: -o user_xattr,acl,flock mft01@tcp:/lustre /mnt/lustre2
debug=-1
subsystem_debug=0xffb7e3ff
debug_mb=2
 - unlinked 0 (time 1331863019 ; total 0 ; last 0)
total: 50 unlinks in 0 seconds: inf unlinks/second
Resetting fail_loc on all nodes...done.
PASS 0a (93s)
.......................== replay-dual replay-dual.sh test complete, duration 106 sec == 03:56:59 (1331863019)
Stopping clients: mft01 /mnt/lustre2 (opts:)
Stopping client mft01 /mnt/lustre2 opts:
